proto/sge/reward/query.proto

```diff
@@ -45,19 +45,19 @@ service Query {
     option (google.api.http).get = "/sge-network/sge/reward/rewards/{address}";
   }
 
-  // Queries list of all Reward items by user address and reward category
+  // Queries list of all Reward items by user address and reward type
   // endpoint.
-  rpc RewardsByAddressAndCategory(QueryRewardsByAddressAndCategoryRequest)
-      returns (QueryRewardsByAddressAndCategoryResponse) {
+  rpc RewardsByAddressAndType(QueryRewardsByAddressAndTypeRequest)
```

Comment:
Category is the right word in this case.

If you see the difference between type and category you will understand that we will need category query here